src/main/java/com/minecolonies/coremod/placementhandlers/main/SurvivalHandler.java

```diff
@@ -66,27 +75,7 @@ public Component getDisplayName()
     public boolean canHandle(final Blueprint blueprint, final ClientLevel clientLevel, final Player player, final BlockPos blockPos, final PlacementSettings placementSettings)
     {
         BlockState blockState = blueprint.getBlockState(blueprint.getPrimaryBlockOffset());
-        if (blockState.is(ModBlocks.blockPlantationField))
-        {
-            return false;
-        }
-
-        if (blueprint.getBlockState(blueprint.getPrimaryBlockOffset()).getBlock() instanceof BlockHutTownHall)
-        {
-            return true;
-        }
-
-        final IColonyView colonyView = IColonyManager.getInstance().getClosestColonyView(clientLevel, blockPos);
-        if (colonyView == null)
-        {
-            return false;
-        }
-        if (!colonyView.getPermissions().hasPermission(player, Action.ACCESS_HUTS))
-        {
-            return false;
-        }
-
-        return colonyView.isCoordInColony(clientLevel, blockPos);
+        return !blockState.is(ModBlocks.blockPlantationField);
```

Comment:
Isn't this going to make the "assign to builder" button come up even when outside any colony (or inside a colony you don't have permission to build), which wasn't intended?

It should probably still keep these checks here even if it does a more comprehensive check later.

Reply:
This also begs the question of why there's a special `PlantationFieldPlacementHandler` and whether it needs similar changes.